backend/deploy.js

```diff
@@ -34,4 +34,41 @@ exports.deployPages = function (socket, data) {
       socket.emit('deploy-success', {pagesURL: "https://" + data.username + ".github.io/" + repoName});
     }
   });
-}
+};
+
+exports.deployHeroku = function (socket, data) {
+  var spawn = require('child_process').spawn;
+
+  var email = data.email;
+  var herokuAPIKey = data.herokuAPIKey;
```

Comment:
would it be possible to get the API key similar to how we obtain github's oauth token instead of asking the user to input?